srcMain/main.cpp

```diff
@@ -839,6 +839,24 @@ void shasta::main::assemble(
         assembler.writeOrientedReadsByAssemblyGraphEdge();
     }
 
+    string peakMemoryUsage;
```

Comment:
Make this block into a separate function `getPeakMemoryUsage`? It could go in `platformDependent.hpp` and `platformDependent.cpp` or in its own pair of `.hpp`, `.cpp` files.